pkg/migrations/duplicate_test.go

```diff
@@ -67,35 +68,37 @@ func TestDuplicateStmtBuilderCheckConstraints(t *testing.T) {
 		},
 		"single-column check constraint with single column duplicated": {
 			columns:       []string{"email"},
-			expectedStmts: []string{`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_email_at" "_pgroll_new_email" ~ '@' NOT VALID`},
+			expectedStmts: []string{`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_email_at" CHECK ("_pgroll_new_email" ~ '@') NO INHERIT NOT VALID`},
 		},
 		"multiple multi and single column check constraint with single column duplicated": {
 			columns: []string{"age"},
 			expectedStmts: []string{
-				`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_adults" "_pgroll_new_age" > 18 NOT VALID`,
-				`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_new_york_adults" "city" = 'New York' AND "_pgroll_new_age" > 21 NOT VALID`,
+				`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_adults" CHECK ("_pgroll_new_age" > 18) NOT VALID`,
+				`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_new_york_adults" CHECK ("city" = 'New York' AND "_pgroll_new_age" > 21) NOT VALID`,
 			},
 		},
 		"multiple multi and single column check constraint with multiple column duplicated": {
 			columns: []string{"age", "description"},
 			expectedStmts: []string{
-				`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_adults" "_pgroll_new_age" > 18 NOT VALID`,
-				`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_new_york_adults" "city" = 'New York' AND "_pgroll_new_age" > 21 NOT VALID`,
+				`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_adults" CHECK ("_pgroll_new_age" > 18) NOT VALID`,
+				`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_new_york_adults" CHECK ("city" = 'New York' AND "_pgroll_new_age" > 21) NOT VALID`,
 			},
 		},
 		"multi-column check constraint with multiple columns with single column duplicated": {
 			columns:       []string{"name"},
-			expectedStmts: []string{`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_different_nick" "_pgroll_new_name" != "nick" NOT VALID`},
+			expectedStmts: []string{`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_different_nick" CHECK ("_pgroll_new_name" != "nick") NOT VALID`},
 		},
 		"multi-column check constraint with multiple columns duplicated": {
 			columns:       []string{"name", "nick"},
-			expectedStmts: []string{`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_different_nick" "_pgroll_new_name" != "_pgroll_new_nick" NOT VALID`},
+			expectedStmts: []string{`ALTER TABLE "test_table" ADD CONSTRAINT "_pgroll_dup_different_nick" CHECK ("_pgroll_new_name" != "_pgroll_new_nick") NOT VALID`},
 		},
 	} {
 		t.Run(name, func(t *testing.T) {
 			stmts := d.duplicateCheckConstraints(nil, testCases.columns...)
 			assert.Equal(t, len(testCases.expectedStmts), len(stmts))
 			for _, stmt := range stmts {
+				_, err := pgq.Parse(stmt)
+				assert.NoError(t, err)
```

Comment:
What's the reason for this change?

Reply:
The statements that were generated previously had SQL syntax errors. It was never a problem for pgroll, it never caused a bug. But I expected that this statement generator produces a valid SQL statement.